otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css

```diff
@@ -404,6 +407,14 @@ h5 {
   font-weight: lighter;
 }
 
+h1, h2, h3 {
+  margin-bottom: 1rem;
+}
+
+.docs-content h2, h3, h4 {
+  margin-top: 3rem;
```

Comment:
this is too much, it leaves huge gaps. 2rem is ok

Reply:
done